.env

```
@@ -0,0 +1,3 @@
+TODO_DBFILE="../cmd/scheduler.db"
```

Вообще проще запускать проект из корня командой `go run cmd/main.go`, поэтому путь лучше делать от корня проекта

---

Так же БД можно хранить в корне проекта
```suggestion
TODO_DBFILE="./scheduler.db"
```